core/src/main/java/edu/wpi/grip/core/sources/CameraSource.java

```diff
@@ -165,6 +169,25 @@ public String getName() {
     }
 
     @Override
+    protected boolean updateOutputSockets() {
```

Comment:
JavaCV's `FrameGrabber` API works by polling.  If we're changes sources to poll instead of using events, we should just call the JavaCV functions here instead of running it in a separate thread